src/qibocal/tree/test.py

```diff
@@ -1,7 +1,33 @@
+from queue import Queue
+
+import pytest
+from task import Task
+
 # Each task is a list
 # [name, parameters, requirements]
 tasks = [
-    ["command1", None, None],
-    ["command2", None, "command1"],
-    ["command3", None, "command2"],
+    ["command_1", [], ["start"]],
+    ["command_2", [], ["command_1"]],
+    ["command_3", [], ["command_2", "command_1"]],
 ]
+
+
+def test_queue():
+    print(tasks)
+    list = [Task.load(i) for i in tasks]
+    print(type(list[0]))
+    q = Queue(list)
+    print("prova", q.queue)
+    ready = q.free()  # TODO: fix AttributeError: 'Queue' object has no attribute 'free
+    assert ready[0].requirements == {"start": True}
+    assert ready[0].ready == True
+
+
+def test_Task():
```

Comment:
```suggestion
def test_task():
```
Please, respect Python conventions on identifiers case
https://peps.python.org/pep-0008/